### ISSUE TITLE
Longbow does not work with Valve Index Controller

### ISSUE THREAD

**Issue by elwes** [ValveSoftware/steamvr_unity_plugin#694]
While testing around with different controllers I noticed, that the longbow in the demo scene can not be fired when used with the index controllers. The vive controllers work just fine. Can someone confirm?

Steps to reproduce: Load SteamVR PlugIn 2.0, find the demo scene with the longbow and try to fire the arrow with the index controllers. You can pick up the longbow, you will get the arrow but you can not use the arrow with the longbow.

**Comment by Giantbean** [comment]
It can be fired but you have to have tour thumb on the track pad and the trigger held down. Very awkward at first but doable.

**Comment by elwes** [comment]
Yes, works! Now it would be nice to get it working without holding down down the trackpad. Thanks for your response!

**Comment by Arch1medes** [comment]
@elwes, This functionality is due to the grab type being a "Pinch", which can be adjusted in the Bow's code. A Pinch grab is performed with the top of the pill pad + the trigger. More info can be found [here](https://valvesoftware.github.io/steamvr_unity_plugin/articles/Interaction-System.html) under the "Hand" Section.

**Comment by Waffle1434** [comment]
This is an extremely unintuitive gesture.
